frontend/views/components/Modal/ModalTemplate.vue

```diff
@@ -137,7 +137,7 @@ export default {
 }
 
 .c-modal-body {
-  margin: $spacer $spacer 2.5rem $spacer;
+  margin: $spacer $spacer $spacer-lg $spacer;
 
   &:last-child {
     padding-bottom: $spacer-lg;
```

Comment:
I noticed we have 4 modifiers for modals:
- .has-no-background - not being used. Maybe it's the default state and this can be deleted.
- .has-background - only used on DS example...
- .has-background-footer - what's the difference of this to `.has-background`?
- .has-submodal-background - when applied it doesn't add any background even on mobile, maybe we should give it a better name?